src/method.c

```diff
@@ -631,6 +656,26 @@ JL_DLLEXPORT jl_method_t *jl_new_method_uninit(jl_module_t *module)
 
 // method definition ----------------------------------------------------------
 
+jl_method_t *jl_make_opaque_closure_method(jl_module_t *module,
+    jl_value_t *nargs, jl_value_t *functionloc, jl_code_info_t *ci)
+{
+    jl_method_t *m = jl_new_method_uninit(module);
+    JL_GC_PUSH1(&m);
+    // TODO: Maybe have a signature of (parent method, stmt#)?
```

Comment:
Yeah, but for now, I think this just needs an explicit check in the serializers (dump and Serialization.jl) so they bail explicitly, and don't try to map it to a non-existent method with a signature of Tuple.